### ISSUE TITLE
How to get whether the storage account enable secure transfer

### ISSUE THREAD

**Issue by gavinfish** [Azure/azure-storage-java#402]
### Which service(blob, file, queue, table) does this issue concern?

all of them

### Which version of the SDK was used?

6.1.0

### What problem was encountered?
I want to get whether a storage account enables secure transfer(https) by the sdk

**Comment by rickle-msft** [comment]
Hi, @gavinfish. HTTPS should always work with Storage. If you want to check whether your account is set to accept https only, you have to use the management plane apis. If you want to check whether the storage sdk is making calls using https, you'll have to check the protocol for the passed enpoint or connection string.

**Comment by gavinfish** [comment]
Thanks @rickle-msft , do you mean even users disabled the `Secure transfer required` feature in their configuration, I can still use HTTPS. So the codes below could communicate with any storage accounts successfully, right?
```
new CloudStorageAccount(
                    new StorageCredentialsAccountAndKey(storageAccount.name(), storageAccountKey),
                    true,// useHttps
                    blobSuffix);
```

**Comment by rickle-msft** [comment]
@gavinfish Correct. `Secure transfer required` only changes HTTP behavior by accepting/rejecting it if secure transfer is required/not required respectively. There is not a way, as far as I know, to actually disallow HTTPS. HTTPS should always be accepted.
